packages/react/src/Link/Link.module.css

```diff
@@ -1,4 +1,4 @@
-:where(.Link) {
+.Link {
```

Comment:
Is there any risk in this change or should be good to go? 👀

Reply:
We should integration test this I found there was another error in Timeline with `:where` here.